.github/workflows/build-with-emsdk.yml

```diff
@@ -0,0 +1,87 @@
+name: Build emscripten tools using emsdk
+
+on:
+  workflow_dispatch:
+    inputs:
+      llvm_build:
+        description: 'LLVM Arm64 build'
+        required: false
+        default: false
+        type: boolean
+      python_build:
+        description: 'Python Arm64 build'
+        required: false
+        default: false
+        type: boolean
+      node_build:
+        description: 'Node Arm64 build'
+        required: false
+        default: false
+        type: boolean
+      binaryen_build:
+        description: 'Binaryen Arm64 build'
+        required: false
+        default: false
+        type: boolean
+      emscripten_build:
+        description: 'Emscripten Arm64 build'
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  build_llvm:
+    if: ${{ inputs.llvm_build }}
+    
+    runs-on: [self-hosted, Windows, ARM64, WASM]
+    timeout-minutes: 60
```

Comment:
I think it might take up to 6h

Reply:
Yeah, I can imagine, I'll adjust the timeouts on demand in the consecutive PRs.